CHANGELOG.md

```diff
@@ -4,12 +4,28 @@ Please note that this application has changed its name over time.
 It was first called "Evopedia" (and was using the file format of Evopedia).
 Then it was renamed to "Kiwix HTML5" (and used the ZIM file format). Finally it was renamed "Kiwix JS".
 
-## In-progress (interim) Kiwix JS v4.0.1
-
-Note: this is temporarily released as an update to the browser-extension remote PWA only
-
-* FEATURE: Ability optionally to re-open an archive automatically in Chromium browsers > 122
+## Kiwix JS v4.1.0
+
+Released on *TODO*
+
+* FEATURE: Preview a WikiMedia ZIM article in a popover by hovering, long-pressing, or tabbing into a link
+* FEATURE: Automatically re-open the last selected archive in Chromium browsers > 122
+* FEATURE: New security dialogue box when opening untrusted ZIMs now shows enhanced ZIM metadata
+* ENHANCEMENT: Display active content warning when opening PhET ZIMs in Restricted mode
+* ENHANCEMENT: Added option to turn off external link warning
+* UPDATE: JQuery mode is now renamed to Restricted mode to better indicate its effect
+* UPDATE: JQuery is now fully removed from all non-dependency code
```

Comment:
What is "non-dependency code" and how does it differ from "code"?

Reply:
Oh, I just meant JQuery has been removed from all the Kiwix JS code (written by us). However, dependencies like Bootstrap may still use JQuery (in fact the Bootstrap version we're currently using does: there was a PR to update the Bootstrap version #1235, it may yet be completed if conflicts can be resolved). If you can think of a clearer wording, please do!

Reply:
Got it. Yeah that occurred to me at like 11 PM last night as I was going to bed but I wasn't near a computer lol.

I think maybe "removed from first party code" is a little clearer?